packages/site/src/components/Nav.js

```diff
@@ -45,6 +45,9 @@ const styles = {
     margin: '0',
     fontSize: theme.fontSize_h4
   }),
+  link: {
+    textDecoration: 'none'
```

Comment:
Is this needed anymore, following the latest Link update?  https://github.com/mineral-ui/mineral-ui/blob/master/packages/link/src/Link.js#L57